src/universal/components/LinkNewTab/LinkNewTab.js

```diff
@@ -0,0 +1,28 @@
+import React, {PropTypes} from 'react';
+
+const LinkNewTab = (props) => {
+  const { children, href, title } = props;
+  let url = href;
+  // Add http protocol to url if no protocol was provided
+  const urlProtocolRegex = new RegExp(/http:\/\/|https:\/\/|mailto:|ftp:\/\//);
```

Comment:
Talk to me about this @ndey96: why do you feel it's necessary to do this verification and mutation of data? Are you guarding against something?